tap_asana/asana.py

```diff
@@ -30,8 +30,6 @@ def _access_token_auth(self):
     return asana.Client.access_token(self.access_token)
 
   def refresh_access_token(self):
-    if self._client.session.authorized:
-      return
```

Comment:
Removed above condition because this function is only called when the access token expires so no need to return directly without a refreshing token.